src/System.Resources.ReaderWriter/src/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
That's what @weshaggard and @ericstj have been urging for the corefx repo for the time being.